pkg/server/plugin/nodeattestor/azure/msi_test.go

```diff
@@ -139,6 +139,26 @@ func (s *MSIAttestorSuite) TestAttestFailsWithBadSignature() {
 		"unable to verify token")
 }
 
+func (s *MSIAttestorSuite) TestAttestFailsWithAlgorithmMismatch() {
+	s.addKey("KEYID")
+
+	// sign a token with a different key algorithm than that of the key in
+	// the key set.
+	signer, err := jose.NewSigner(jose.SigningKey{
+		Algorithm: jose.HS256,
+		Key:       []byte("0123456789ABCDEF"),
```

Comment:
I think this should be `s.key.Public()`?

Reply:
heh, square/go-jose panics during signing if you do that